### PR TITLE
Set config file name correctly

### PR DESCRIPTION
**Pull request by mereba**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func initConfig(c *cobra.Command, args []string) error {
 		// Use config file from the flag.
 		viper.SetConfigFile(configFile)
 	} else {
-		viper.SetConfigName(".klepto")
+		viper.SetConfigName(".klepto.toml")
```
Comment (italolelis) [inline cmd/root.go]: This is not necessary. You can have a configuration file in any format (toml, yaml, json), viper should handle this for us.

Comment (mereba) [reply cmd/root.go]: Ah, makes sense. Changed config name in the command example and `init()` function.

File: cmd/root.go [hunk 2]
```diff
@@ -27,12 +27,12 @@ var (
...
 	
 		Perfect for bringing your live data to staging!`,
-		Example: "klepto steal -c klepto.toml --from root:root@localhost:3306/example --to root:root@localhost:3306/klepto",
+		Example: "klepto steal -c .klepto --from root:root@localhost:3306/example --to root:root@localhost:3306/klepto",
 	}
 )
 
 func init() {
-	RootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "Path to config file (default is $HOME/.klepto.toml)")
+	RootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "Path to config file (default is $HOME/.klepto)")
```
Comment (boekkooi-fresh) [inline cmd/root.go]: Can use change `$home` with `.`?

Comment (mereba) [reply cmd/root.go]: Done

**Pull request merged**